src/main/java/nl/novi/techiteasy1121/controllers/TelevisionController.java

```diff
@@ -1,40 +1,43 @@
 package nl.novi.techiteasy1121.controllers;
 
-import nl.novi.techiteasy1121.Dtos.TelevisionDto;
-import nl.novi.techiteasy1121.Dtos.TelevisionInputDto;
+import nl.novi.techiteasy1121.dtos.IdInputDto;
+import nl.novi.techiteasy1121.dtos.TelevisionDto;
+import nl.novi.techiteasy1121.dtos.TelevisionInputDto;
+import nl.novi.techiteasy1121.dtos.WallBracketDto;
 import nl.novi.techiteasy1121.services.TelevisionService;
-import org.springframework.beans.factory.annotation.Autowired;
+import nl.novi.techiteasy1121.services.TelevisionWallBracketService;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import jakarta.validation.Valid;
+import java.util.Collection;
 import java.util.List;
 import java.util.Optional;
 
 @RestController
 public class TelevisionController {
 
-    // We importeren hier (via de constructor, maar het mag ook @Autowired zijn) nu de Service in plaats van direct de Repository.
     private final TelevisionService televisionService;
 
+    private final TelevisionWallBracketService televisionWallBracketService;
 
```

Comment:
De controller methoden zijn goed ingericht. Het gebruik van ResponseEntity om de HTTP-responsen te structureren is een goede keuze.